lib/files/cachefiles/cache.go

```diff
@@ -67,51 +61,81 @@ func (h *FileStore) Open(ctx context.Context, uri *url.URL) (files.Reader, error
 
 	filename := trimScheme(uri)
 
-	f, ok := h.cache[filename]
+	ctx, reentrant := isReentrant(ctx)
+	if reentrant {
+		// We are in a reentrant caching scenario.
+		// Continuing will deadlock, so we won‘t even try to cache at all.
```

Comment:
```suggestion
		// Continuing will deadlock, so we won’t even try to cache at all.
```

Ugh, extended keyboards that put „“” on the keys VBN, rather than XVB.